### PR TITLE
Markdown README

### PR DESCRIPTION
**Pull request by judy2k**
Implementing the README template from [client-library-specification](https://github.com/Nexmo/client-library-specification/blob/master/templates/README.md)

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -0,0 +1,110 @@
...
+Alternatively you can clone the repo and build the jar yourself:
+
+```bash
+git clone git@github.com:nexmo/nexmo-java.git
+gradle build
+```
+
+## Usage
+
+Check the Javadoc for full documentation. In the meantime, check out the example below:
```
Comment (leggetter) [inline README.md]: Is there a link for this?

File: README.md [hunk 2]
```diff
@@ -0,0 +1,110 @@
...
+
+## Example
+
+### Sending A Message
+
+Use [Nexmo's SMS API][doc_sms] to send an SMS message.
```
Comment (leggetter) [inline README.md]: Nitpick - we tend to avoid apostrophes. So, Use the `Nexmo SMS API`

File: README.md [hunk 3]
```diff
@@ -0,0 +1,110 @@
...
+
+```java
```
Comment (leggetter) [inline README.md]: I would either add a full set of examples or remove this.

File: README.md [hunk 4]
```diff
@@ -0,0 +1,110 @@
+# Nexmo Client Library for Java
+
+You can use this [LANGUAGE] client library to add [Nexmo's API](#api-coverage) to your application. To use this, you'll
+need a Nexmo account. Sign up [for free at nexmo.com][signup].
+
+ * [Installation](#installation)
+ * [Configuration](#configuration)
+ * [Usage](#usage)
+ * [Examples](#examples)
+ * [Coverage](#api-coverage)
```
Comment (leggetter) [inline README.md]: I'd add `API` to this. I thought it meant "Test Coverage". If it says this in the spec then maybe we should update there?

File: README.md (hunk 1 above)
Comment (judy2k) [inline README.md]: Not yet. I think we need to work out a plan for hosting the Javadoc. I'm throwing a doc together right now that includes this fact :smile:

File: README.md (hunk 3 above)
Comment (judy2k) [inline README.md]: I'd like to add more examples before release, just not in this PR.

**Comment by judy2k** [comment]
These changes have been incorporated into the `version-2` branch, so I'm going to close this PR.

**Pull request closed without merging**